### DISCUSSION TITLE
Execute raw transaction using an Aptos Module?

### DISCUSSION THREAD

**Discussion by angieyth** [aptos-labs/aptos-developer-discussions#173] [Questions] [answered]
### Describe your question in detail.

is there any way to execute raw transactions using an aptos module? I want to pass in a raw transaction (something like serializedTransaction: Vec<u8>) as an argument in a function.

### What have you tried or looked at?  Or how can we reproduce the error?

N/A

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by gregnazario** [comment, marked as answer]
No, currently there is no way to pass in a transaction as an input to another transaction.  The closest would be governance, where a hash is stored on-chain for later execution (with the same hash).  See:

https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/aptos_governance.move#L332-L343
